### PR TITLE
Add quarterly roundup for Q3 2022

### PR DESCRIPTION
**Pull request by chrean**
## Summary

Just adding quarterly Roundup, Q3 2022. Better late...

## Checklist

- [x] I have written a thorough PR description.
- [x] I have kept my commits small and atomic.
- [x] I have used clear, explanatory commit messages.

### CODE REVIEW

File: source/blog/2022-11-17-solidus-quarterly-roundup-q3-2022.html.markdown [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+
+This is Gabriel writing, Head of R&D at Nebulab in my spare time, and now contributing to Solidus! So excited to have joined the team and share what has been accomplished last quarter and what plans the future holds!
+
+I am going to pick this up where  left it - no worries, Ale is not going anywhere, it’s just the team growing!
+
+
+# What’s new
+
+
+# Solidus 3.2 is out!
```
Comment (kennyadsl) [inline source/blog/2022-11-17-solidus-quarterly-roundup-q3-2022.html.markdown:20]: Might be better as an h2?

File: source/blog/2022-11-17-solidus-quarterly-roundup-q3-2022.html.markdown [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+
+We officially released the new guides we have been working on over the last year: you can now access them at [https://guides.solidus.io](https://guides.solidus.io)! Find the corresponding repository here: [https://github.com/solidusio/edgeguides](https://github.com/solidusio/edgeguides).
+
+As you can see, we used [Docusaurus](https://docusaurus.io/) as the engine powering the guides. It’s a React-based project that will help focus on writing good content while providing enough flexibility for customizing how the guides are accessed by the community.
+
+In terms of content, we decided to adopt [Diátaxis](https://diataxis.fr/), a framework that will help us structure technical documentation based on the need of the people consuming it. Do you want to see it in action? Check out one of the first pages we converted using the framework: [State Machines](https://guides.solidus.io/advanced-solidus/state-machines/).
+
+Please let us know what you think about them, and what you think can be improved! And contributions are very welcome as usual.
+
+Oh, in case you still need the old guides, they have not been deleted but just moved to https://legacy-guides.solidus.io/.
```
Comment (kennyadsl) [inline source/blog/2022-11-17-solidus-quarterly-roundup-q3-2022.html.markdown:73]: `https://legacy-guides.solidus.io/` might also be a link.

File: source/blog/2022-11-17-solidus-quarterly-roundup-q3-2022.html.markdown [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+author: Gabriel Sambarino
+cover_image: /blog/shared/images/solidus-quarterly-roundup.png
+description: Find out what happened in the Solidus ecosystem in Q3 2022!
+---
+
+Hello again, Solidus community!
+
+This is Gabriel writing, Head of R&D at Nebulab in my spare time, and now contributing to Solidus! So excited to have joined the team and share what has been accomplished last quarter and what plans the future holds!
+
+I am going to pick this up where  left it - no worries, Ale is not going anywhere, it’s just the team growing!
```
Comment (kennyadsl) [inline source/blog/2022-11-17-solidus-quarterly-roundup-q3-2022.html.markdown:14]: Probably a full `Alessandro` is better here.

File: source/blog/2022-11-17-solidus-quarterly-roundup-q3-2022.html.markdown [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+
+The new starter frontend is here to make our beloved users’ lives easier.
+
+On fresh Solidus installations, this modern and easy-to-extend codebase will provide a more efficient bootstrapping experience.
+
+The biggest difference with the old frontend is that the new one does not come as a gem: it is in fact installed via a command that creates all the necessary files inside of the project instead.
+
+This new approach has the primary benefit of not requiring Deface for customizations, thus making developers’ lives easier.
+
+Plus, following the Rails ideology, the new Starter Frontend is opinionated enough to streamline potentially tough decisions for you, while maintaining all the freedom that makes Solidus a great choice for any brand in need of total flexibility. In its current state, the new frontend should be considered just as a starting point, though. In fact, we plan a lot of changes to make it more in line with Rails developers’ expectations: the good news is that we can change it quite easily because it is no more a dependency hence no risk of breaking any backward compatibility.
```
Comment (kennyadsl) [inline source/blog/2022-11-17-solidus-quarterly-roundup-q3-2022.html.markdown:55]: `Rails ideology` -> I'd call it `The Rails Doctrine`, which is actually a specific thing, and we could also link it: https://rubyonrails.org/doctrine

**Review by kennyadsl — COMMENTED** [review]
Left some small suggestions in line. Let me also add:

- The structure of titles seems a bit off. There are H1 into H1, which I guess we can convert into H2 and so on.
- [nice to have] Having some images/screenshots for paragraphs might improve the readers' experience. If you think it's a good idea, I can help find something for them.

Thanks @chrean!

File: source/blog/2022-11-17-solidus-quarterly-roundup-q3-2022.html.markdown (hunk 1 above)
Comment (chrean) [reply source/blog/2022-11-17-solidus-quarterly-roundup-q3-2022.html.markdown]: Makes sense, e26a3e6

File: source/blog/2022-11-17-solidus-quarterly-roundup-q3-2022.html.markdown (hunk 4 above)
Comment (chrean) [reply source/blog/2022-11-17-solidus-quarterly-roundup-q3-2022.html.markdown]: Fixed, 9bfaf20

**Comment by chrean** [comment]
> [nice to have] Having some images/screenshots for paragraphs might improve the readers' experience. If you think it's a good idea, I can help find something for them.

That would be awesome, thank you! Let me know if you need some help with anything!

**Comment by chrean** [comment]
Also fixed header formats.

**Comment by chrean** [comment]
Pictures added, locally tested, and working as expected, I think we should be ready to go.

**Review by kennyadsl — APPROVED** [review]

**Pull request merged**